k8s/nginx-ingress/prod-platform-ingress-resource.yaml

```diff
@@ -3,6 +3,33 @@ kind: ConfigMap
 data:
   proxy-body-size:  "200m"
   client-max-body-size: "200m"
+  location-snippets: "
+    auth_request /auth;
+    error_page 500 = @internal_server_error;
+    error_page 404 = @page_not_found;
+    error_page 403 = @access_denied;
+    error_page 401 = @unauthorized;"
+  server-snippets: "
+    location @internal_server_error { return 500 'Something went wrong!'; }
+    location @page_not_found { return 404 'Page not found :('; }
+    location @access_denied { return 403 'Access denied :|'; }
+    location @unauthorized { return 401 'Unauthorized'; }
+
+    location = /auth {
+      if ($request_uri = '/') { return 200; }
+      if ($request_uri = '/login') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/loginUser)|(/api/v2/users/loginUser)') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/forgotPassword)|(/api/v2/users/forgotPassword)') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/candidates/register)|(/api/v2/users/candidates/register)') { return 200; }
+      if ($request_uri ^ '/static/') { return 200; }
```

Comment:
Publicly availing the Netmanager static files

Reply:
Brilliant! thanks @123MwanjeMike